src/checks_gen.rs

```diff
@@ -887,6 +891,7 @@ impl CheckCodePrefix {
             CheckCodePrefix::B0 => PrefixSpecificity::Hundreds,
             CheckCodePrefix::B00 => PrefixSpecificity::Tens,
             CheckCodePrefix::B002 => PrefixSpecificity::Explicit,
+            CheckCodePrefix::B006 => PrefixSpecificity::Explicit,
```

Comment:
@harupy - BTW -- any thoughts on the approach I took to this problem? I.e., code-generating the prefix map, to allow static enums everywhere?

Reply:
How does flake8 implement this?